setup.py

```diff
@@ -63,7 +63,7 @@
 ]
 
 setup_requires = ['numpy>=' + astropy.__minimum_numpy_version__]
-install_requires = ['numpy>=' + astropy.__minimum_numpy_version__]
+install_requires = ['pytest', 'numpy>=' + astropy.__minimum_numpy_version__]
```

Comment:
Should this be ``pytest>2.8`` instead of just ``pytest``?  The `minversion` in `setup.cfg` I think is not known by `setup.py` so e.g. a pip install won't update it as needed.